[Bug]: typeutil UT fails due to error string in-consistent

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:af46998
- Deployment mode(standalone or cluster): both
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```


### Current Behavior

Unit test fails with 
```
--- FAIL: TestSchema_invalid (0.00s)
    --- FAIL: TestSchema_invalid/field_not_exist (0.00s)
        schema_test.go:286: 
            	Error Trace:	schema_test.go:286
            	Error:      	Error message not equal:
            	            	expected: "Failed to get primary key field: no primary in schema"
            	            	actual  : "failed to get primary key field: no primary in schema"
            	Test:       	TestSchema_invalid/field_not_exist
```

### Expected Behavior

Unit test passed

### Steps To Reproduce

```markdown
Run unit test
```


### Anything else?

_No response_

--- a/internal/util/typeutil/schema_test.go
+++ b/internal/util/typeutil/schema_test.go
@@ -283,7 +283,7 @@ func TestSchema_invalid(t *testing.T) {
 
 		_, err = helper.GetPrimaryKeyField()
 		assert.NotNil(t, err)
-		assert.EqualError(t, err, "Failed to get primary key field: no primary in schema")
+		assert.EqualError(t, err, "failed to get primary key field: no primary in schema")
 
 		_, err = helper.GetFieldFromName("none")
 		assert.NotNil(t, err)

